cypress/support/commands.js

```diff
@@ -1,12 +1,12 @@
 import detectSound from './detectSound';
 
 Cypress.Commands.add('joinRoom', (username, roomname) => {
-  // These tests were written before Grid View was implemented. This app now activates
-  // Grid View by default, so here we activate Presentation View before visiting the app so
+  // These tests were written before Gallery View was implemented. This app now activates
+  // Gallery View by default, so here we activate Presentation View before visiting the app so
```

Comment:
```suggestion
  // Gallery View by default, so here we activate Speaker View before visiting the app so
```
Might be worth it to search all files in VS Code for "grid" and "presentation"

Reply:
Thanks, looks like that should be all!